build

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
same in this script please:
- set -xe
- no need to escape `/` for sed regex
- move `*.jar` only